test/core/end2end/tests/bad_ping.cc

```diff
@@ -66,17 +66,13 @@ static void end_test(grpc_end2end_test_fixture* f) {
 static void test_bad_ping(grpc_end2end_test_config config) {
   grpc_end2end_test_fixture f = config.create_fixture(nullptr, nullptr);
   cq_verifier* cqv = cq_verifier_create(f.cq);
-  grpc_arg client_a[3];
+  grpc_arg client_a[2];
   client_a[0].type = GRPC_ARG_INTEGER;
```

Comment:
Might as well change all of these to use `grpc_channel_arg_integer_create()` while we're in here anyway.

Same thing throughout all of the tests in this PR.